### PR TITLE
Preserve comments in XML unless ooxmlViewer.preserveComments is false

### LINKED ISSUE #37: Comments not preserved in XML parts
I'm using version `1.2.2` on Windows 10. When I edit a part of an OOXML package and add a comment liek the following: 

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<calcChain xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
  <!-- <c r="C1" i="1" l="1"/> -->
  <c r="C1" i="1" l="1"/>
</calcChain>
```

Upon save (`Ctrl-S` in VS Code), the part is saved but the commented line disappears. 

Is it possible to preserve the comment?

### PR DESCRIPTION
**Pull request by mikeebowen**
This commit adds `ooxmlViewer.preserveComments` to settings.json. If set false comments will be removed from xml files when the file is saved, otherwise comments in XML are preserved. This closes #37 .

### CODE REVIEW

File: src/extension-utilities.ts [hunk 1]
```diff
@@ -79,7 +79,9 @@ export class ExtensionUtilities {
    * @param {boolean} preserveComments
```
Comment (yuenm18) [inline src/extension-utilities.ts:79]: Can you remove this from the comment block

Comment (mikeebowen) [reply src/extension-utilities.ts]: Done. Good catch I missed that.

**Review by yuenm18 — APPROVED** [review]

**Comment by mikeebowen** [comment]
@yuenm18 Are you going to do a release for this?

**Comment by yuenm18** [comment]
Yea, I'll do it tonight

**Comment by mikeebowen** [comment]
Awesome, thanks

**Pull request merged**